app/services/entry_service.py

```
@@ -95,6 +95,94 @@ def _refresh_entry_date(self, entry: Entry) -> None:
+                        user_id=str(user_id),
+                        media_id=str(media.id),
+                        file_path=media.file_path
+                    )
+
+                    # Delete thumbnail if it exists
+                    if media.thumbnail_path:
+                        try:
+                            full_thumbnail_path = (media_root / media.thumbnail_path).resolve()
+                            if full_thumbnail_path.exists() and str(full_thumbnail_path).startswith(str(media_root)):
+                                full_thumbnail_path.unlink(missing_ok=True)
+                        except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)